### DISCUSSION TITLE
Tilten image after upgrade

### DISCUSSION THREAD

**Discussion by tomasenskede** [jomjol/AI-on-the-edge-device#2991] [General]
### The Problem

after update to:

Firmware version: | Release: v15.7.0 (Commit: 0d0b018+)
Firmware build time: | 17.02.2024 00:15
Web interface version: | Release: v15.7.0 (Commit: 0d0b018+)

Why tiltet?

Ref image is ok!

### Version

15.7

### Logfile

```shell
2
```

### Screenshots

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/32595351/8e07a1c0-ac1e-43d4-8d39-ec76a004f599)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/32595351/bfb81b30-76d5-4dcb-a58b-49522cbe47ac)

**Comment by SybexX** [comment]
Your Alignment Markers are apparently not being recognized correctly
https://jomjol.github.io/AI-on-the-edge-device-docs/Alignment/

**Comment by caco3** [comment]
the marks are too large and too close to each other
